### PR TITLE
Add the negative test cases for test_config.py

### PR DESCRIPTION
**Pull request by hkx303**
**Description**

- [x] test_debug_in_virtwho_conf
- [x] test_oneshot_in_virtwho_conf
- [x] test_reporter_id_in_virtwho_conf
- [x] test_configs_in_virtwho_conf
- [x] test_owner_in_virtwho_conf
- [x] test_hypervisor_id_in_virtwho_conf

**Test Result**
```
[hkx303@kuhuang virtwho-test]$ pytest tests/function/test_config.py::TestConfigurationNegative
==================================== test session starts =====================================
platform linux -- Python 3.7.3, pytest-5.4.3, py-1.8.0, pluggy-0.13.1
rootdir: /home/hkx303/Documents/CI/virtwho-test, inifile: pytest.ini
plugins: services-1.3.1, mock-1.10.4, forked-1.0.2, ibutsu-1.0.34, cov-2.7.1, xdist-1.34.0
collected 6 items                                                                            

tests/function/test_config.py ......                                                   [100%]
========================= 6 passed, 3 warnings in 482.57s (0:08:02) ==========================
```

### CODE REVIEW

File: tests/function/test_config.py [hunk 1]
```diff
@@ -614,3 +615,272 @@ def test_interval_in_virtwho_sysocnfig(self, virtwho, function_sysconfig):
...
+
+        :title: virt-who: config: test oneshot negative option
+        :id: 843f67cb-c98e-4074-9527-ae216fb036df
+        :caseimportance: High
+        :tags: tier2
+        :customerscenario: false
+        :upstream: no
+        :steps:
+
+            1. Run virt-who with "oneshot=True" in /etc/virt-who.conf
```
Comment (Junefen) [inline tests/function/test_config.py:698]: should be `oneshout=[null value]`

File: tests/function/test_config.py [hunk 2]
```diff
@@ -614,3 +615,272 @@ def test_interval_in_virtwho_sysocnfig(self, virtwho, function_sysconfig):
...
+                    and result['thread'] == 1
+                    and result['reporter_id'] == non_ascii)
+
+    @pytest.mark.tier2
+    @pytest.mark.notLocal
+    def test_configs_in_virtwho_conf(self, virtwho, globalconf, hypervisor_data, ssh_host,
+                                     configs_data):
+        """Test the configs option in /etc/virtwho.conf
+
+        :title: virt-who: config: test configs option
```
Comment (Junefen) [inline tests/function/test_config.py:770]: please add the `negative` key word

File: tests/function/test_config.py [hunk 3]
```diff
@@ -614,3 +615,272 @@ def test_interval_in_virtwho_sysocnfig(self, virtwho, function_sysconfig):
...
+        """
+        globalconf.update('global', 'debug', 'True')
+        globalconf.update('defaults', 'hypervisor_id', 'uuid')
+
+        result = virtwho.run_service()
+        assert (result['error'] == 0
+                and result['send'] == 1
+                and result['thread'] == 1
+                and result['hypervisor_id'] == hypervisor_data['hypervisor_hostname'])
+        if REGISTER == 'rhsm':
```
Comment (Junefen) [inline tests/function/test_config.py:883]: it seems no need to delete the hosts info from register server side.

File: tests/function/test_config.py [hunk 4]
```diff
@@ -614,3 +615,272 @@ def test_interval_in_virtwho_sysocnfig(self, virtwho, function_sysconfig):
...
+        ret, output = ssh_host.runcmd(cmd)
+        assert (ret == 0
+                and output is not None
+                and output != ""
+                and "-rw-------" in output)
+
+@pytest.mark.usefixtures('function_globalconf_clean')
+@pytest.mark.usefixtures('class_hypervisor')
+@pytest.mark.usefixtures('class_virtwho_d_conf_clean')
+class TestConfigurationNegative:
```
Comment (Junefen) [inline tests/function/test_config.py:661]: let's make it uniform to add the `positive` and `negative` keyword, my suggestion is
`:title: virt-who: config: test debug option (positive)`
`:title: virt-who: config: test debug option (negative)`

File: tests/function/test_config.py (hunk 3 above)
Comment (hkx303) [reply tests/function/test_config.py:883]: also move this from old cases, that would be great :)

File: tests/function/test_config.py (hunk 2 above)
Comment (hkx303) [reply tests/function/test_config.py:770]: added

File: tests/function/test_config.py (hunk 1 above)
Comment (hkx303) [reply tests/function/test_config.py:698]: Yep, copied it from the positive case, forgot to update it :)

**Pull request merged**